business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -187,23 +187,16 @@
 
   <profiles>
     <profile>
-      <id>kie-drools-wb</id>
-      <properties>
-        <app.name>kie-drools-wb</app.name>
-      </properties>
-    </profile>
-
-    <profile>
-      <id>kie-wb</id>
+      <id>business-central</id>
       <properties>
         <app.name>kie-wb</app.name>
```

Comment:
Please update to `<app.name>business-central</app.name>` on this line

Reply:
we need to follow up with the kie-jenkins-scripts profile changes too when this is merged, so please notify @mbiarnes and @pszubiak